examples/demo/Extras/LED_display.py

```diff
@@ -11,24 +11,16 @@
 using a simulated LED display control.
 """
 
-from threading  import Thread
-
+from threading import Thread
 from time import sleep
 
 from traits.api import HasTraits, Instance, Int, Bool, Float
 
-from traitsui.api import View, Item, HGroup, Handler, UIInfo, spring
-
-from traits.etsconfig.api import ETSConfig
-if ETSConfig.toolkit == 'wx':
-    from traitsui.wx.extra.led_editor import LEDEditor
-else:
-    from traitsui.qt4.extra.led_editor import LEDEditor
```

Comment:
Shall we keep `traitsui.qt4.extra.led_editor` and `traitsui.wx.extra.led_editor` as aliases? It costs almost nothing to maintain aliases. Otherwise this would be a breaking change.